### PR TITLE
docs(with-nextjs): added the guide for using next with chakra ui

### PR DESCRIPTION
**Pull request by Mzaien**
a guide with chakra-ui v1, v0.8 with nextjs

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes
      /start features)

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by Mzaien** [comment]
should i do something to fix that ? @with-heart

**Comment by with-heart** [comment]
I committed a change that should prevent the `public/` folder for showing up in your commit, if you could kindly rebase onto `develop`.

I'll have to look into the scripts a little more later to fix the `.all-membersrc` issue. It's a problem on our end, not something you've done :)

**Comment by Mzaien** [comment]
Done 👍

**Comment by with-heart** [comment]
Not sure what was going on with your branch but I fixed it with a force-push without thinking about it, so your local branch is going to be messed up. Delete it locally and then check out `docs/nextjs-guide` from your repo again and everything will be back to normal.

File: website/pages/guides/integrations/with-nextjs.mdx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+In Next, inside the pages folder of your project is `_app.js`
+
+```js live=false
+import { ChakraProvider, CSSReset } from "@chakra-ui/core"
+import theme from "@chakra-ui/theme"
+function MyApp({ Component, pageProps }) {
+  return (
+    <ChakraProvider theme={theme}>
+      <CSSReset />
```
Comment (with-heart) [inline website/pages/guides/integrations/with-nextjs.mdx:46]: `ChakraProvider` now uses the default theme by default, and has a `resetCSS` prop that adds `CSSReset` automatically.

```suggestion
import { ChakraProvider } from "@chakra-ui/core"
function MyApp({ Component, pageProps }) {
  return (
    <ChakraProvider>
```

File: website/pages/guides/integrations/with-nextjs.mdx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  return (
+    <ThemeProvider>
+      <CSSReset />
+      <Component {...pageProps} />
+    </ThemeProvider>
+  )
+}
+
+export default MyApp
+```
```
Comment (with-heart) [inline website/pages/guides/integrations/with-nextjs.mdx:84]:
```suggestion
```

**Review by with-heart — CHANGES_REQUESTED** [review]
I made some suggestions to try to streamline this a bit :)

File: website/pages/guides/integrations/with-nextjs.mdx (hunk 1 above)
Comment (Mzaien) [reply website/pages/guides/integrations/with-nextjs.mdx:46]: Oh really i did not know , i was on v1.0 rc2

**Comment by Mzaien** [comment]
Followed your instructions, a bare minimum set up with next.js

**Comment by segunadebayo** [comment]
I've added this guide in a recent commit based on the changes in your PR.

Unable to merge due to the number of changed files.

**Pull request closed without merging**